[2022-11-04T19:40:33.033+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: orquestation.TASK1 scheduled__2022-05-02T00:00:00+00:00 [queued]>
[2022-11-04T19:40:33.234+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: orquestation.TASK1 scheduled__2022-05-02T00:00:00+00:00 [queued]>
[2022-11-04T19:40:33.235+0000] {taskinstance.py:1362} INFO - 
--------------------------------------------------------------------------------
[2022-11-04T19:40:33.236+0000] {taskinstance.py:1363} INFO - Starting attempt 1 of 1
[2022-11-04T19:40:33.768+0000] {taskinstance.py:1364} INFO - 
--------------------------------------------------------------------------------
[2022-11-04T19:40:35.412+0000] {taskinstance.py:1383} INFO - Executing <Task(BashOperator): TASK1> on 2022-05-02 00:00:00+00:00
[2022-11-04T19:40:35.419+0000] {standard_task_runner.py:55} INFO - Started process 208 to run task
[2022-11-04T19:40:35.634+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'orquestation', 'TASK1', 'scheduled__2022-05-02T00:00:00+00:00', '--job-id', '3', '--raw', '--subdir', 'DAGS_FOLDER/5.1-orquestacion.py', '--cfg-path', '/tmp/tmp3ha39530']
[2022-11-04T19:40:35.635+0000] {standard_task_runner.py:83} INFO - Job 3: Subtask TASK1
[2022-11-04T19:40:36.460+0000] {task_command.py:376} INFO - Running <TaskInstance: orquestation.TASK1 scheduled__2022-05-02T00:00:00+00:00 [running]> on host 208ed96b478a
[2022-11-04T19:40:39.410+0000] {taskinstance.py:1592} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=orquestation
AIRFLOW_CTX_TASK_ID=TASK1
AIRFLOW_CTX_EXECUTION_DATE=2022-05-02T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-05-02T00:00:00+00:00
[2022-11-04T19:40:41.354+0000] {subprocess.py:63} INFO - Tmp dir root location: 
 /tmp
[2022-11-04T19:40:42.146+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', "SLEEP 2 && ECHO 'TASK1'"]
[2022-11-04T19:40:43.882+0000] {subprocess.py:86} INFO - Output:
[2022-11-04T19:40:43.981+0000] {subprocess.py:93} INFO - /bin/bash: line 1: SLEEP: command not found
[2022-11-04T19:40:43.984+0000] {subprocess.py:97} INFO - Command exited with return code 127
[2022-11-04T19:40:44.057+0000] {taskinstance.py:1851} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/bash.py", line 197, in execute
    f'Bash command failed. The command returned a non-zero exit code {result.exit_code}.'
airflow.exceptions.AirflowException: Bash command failed. The command returned a non-zero exit code 127.
[2022-11-04T19:40:44.090+0000] {taskinstance.py:1406} INFO - Marking task as FAILED. dag_id=orquestation, task_id=TASK1, execution_date=20220502T000000, start_date=20221104T194033, end_date=20221104T194044
[2022-11-04T19:40:44.535+0000] {standard_task_runner.py:105} ERROR - Failed to execute job 3 for task TASK1 (Bash command failed. The command returned a non-zero exit code 127.; 208)
[2022-11-04T19:40:44.595+0000] {local_task_job.py:164} INFO - Task exited with return code 1
[2022-11-04T19:40:44.713+0000] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
